OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -208,6 +243,25 @@ +(void)onStubActivation:( nullable void(^)(NSURLRequest* request, id<OHHTTPStubs
 ////////////////////////////////////////////////////////////////////////////////
 #pragma mark - Private instance methods
 
+-(BOOL)isEnabled
+{
+    BOOL enabled = NO;
+    @synchronized(self)
```

Comment:
Using `self` instead of `_enabledStateNumber` because that might change and cause multiple locks to be enabled for this property